package.json

```diff
@@ -122,6 +122,7 @@
 		"cross-port-killer": "^1.4.0",
 		"date-fns": "^3.3.1",
 		"electron-squirrel-startup": "^1.0.0",
+		"electron2appx": "^2.1.2",
```

Comment:
I noticed that PocketCasts also uses the `electron2appx` library. I’m surprised that the official guide https://www.electronjs.org/docs/latest/tutorial/windows-store-guide suggests a library and we actually need to use a fork of it.

Reply:
It is odd right? And what are the odds that no one run into the need for disabling code signing when building the `appx`? (_I haven't actually investigated existing issues in the repo, maybe there are open ones in that regard. TODO_)

We need this fork because we need to disable code signing. But it's possible the issue is with us having to disable code signing. That is, we should not have to do that.

The reason we have to do it is that the cert we use for code signing comes from a publisher, `CN='Automattic, Inc.', O='Automattic, Inc.', S=California, C=US` that is different from the one in the Store, `CN=E2E5A157-746D-4B04-9116-ABE5CB928306`. My team has an open task to add processes and tooling around the certificate management, so hopefully we'll discover more about this soon.